cdap-api/src/main/java/co/cask/cdap/api/artifact/PluginConfigurer.java

```diff
@@ -25,7 +26,7 @@
  * This interface provides methods to register plugin usage in a CDAP Program. The registered plugins will be
  * available at the runtime of the Program.
  */
-public interface PluginConfigurer {
+public interface PluginConfigurer extends DatasetConfigurer {
```

Comment:
It is used by AbstractFlowlet, AbstractSpark classes. Until we have support for plugins in both, we need these two interfaces. Also, ApplicationConfigurer extends DatasetConfigurer but not PluginConfigurer.